packages/front-end/components/Experiment/PreLaunchChecklist.tsx

```diff
@@ -373,11 +392,11 @@ export function PreLaunchChecklist({
           checklist: updatedManualChecklistStatus,
         }),
       });
+      await mutateExperiment();
```

Comment:
Moving this here resolves a weird experience on slow connections where the `Loading` state resolves, but the checklist item hasn't yet been updated (because we weren't awaiting the `mutateExperiment` call.